views/error.hbs

```diff
@@ -1,23 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <title>Error Page - IDEA Hacks</title>
     <meta charset="UTF-8">
 
     <!-- GOOGLE FONTS -->
-    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap" rel="stylesheet">
 
     <!-- PERSONAL STYLESHEETS -->
     <link rel='stylesheet' type='text/css' href='/css/error-page.css' />
```

Comment:
Add a link to `style.css` like you did in `confirm.hbs`. Make sure it goes above the link you have rn. I tested it and it shouldn't change anything on the page; I'm just having you add it for consistency across all the pages and in case someone in the future decides they want to use classes from there